elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -300,16 +339,134 @@ class PfePrimaryDetail extends PFElement {
         details: nextDetails
       }
     });
+  } // end _handleHideShow()
 
-    // Set focus to pane
-    const firstFocusableElement = nextDetails.querySelector(
-      'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
-    );
-    if (firstFocusableElement) {
-      firstFocusableElement.focus();
+  /**
+   * A11y features
+   */
+  _isToggle(element) {
+    return element.classList.contains("a11y-toggle");
+  }
+
+  _isPanel(element) {
+    return element.classList.contains("a11y-panel");
+  }
+
+  _allToggles() {
+    return [...this.querySelectorAll(".a11y-toggle")];
+  }
+
+  _allPanels() {
+    console.log([...this.querySelectorAll(".a11y-panel")]);
+    return [...this.querySelectorAll(".a11y-panel")];
```

Comment:
`this._slots.details` is an array of the elements in order, might want to remove `console.log()` as well.

Reply:
> Did some quick usability testing, the tab controls needs to take into account if there's something in `details-nav--header` or `details-nav--footer`, in the main demo page the tab order goes from 1rst component, first item, nav-footer, then 1rst panel.
> https://deploy-preview-1404--happy-galileo-ea79c4.netlify.app/elements/pfe-primary-detail/demo/

thanks @wesruv, I noticed that as well. I'm looking into a solution to take into account the header and footers of the tabs.

Reply:
@wesruv what kind of elements will be allowed to go inside the header and footer sections of the tabs? Will it just be heading tags, buttons, and links?

Reply:
That's what it's looking like now, if it's important that the content is a certain kind of thing, or that it shouldn't be a certain kind of thing we should add some docs to the README for those slots.